cluster/manifests/logging-agent/daemonset.yaml

```diff
@@ -237,6 +241,10 @@ spec:
         hostPath:
           path: /var/log/journal
 
+      - name: log-watcher-config
+        configMap:
+          name: log-watcher-config
```

Comment:
This should be marked as optional and the configmap itself not deployed via CLM. Otherwise any changes that you make will get overwritten every time we roll out something to the clusters, which is probably not something you want.

Reply:
We decided we don't want to have this change persisted after cluster redeployment, at least now. This is more emergency option to mitigate unexpected things.

Reply:
Cluster redeployments happen very often, sometimes multiple times per day. We also don't notify anyone for most of them. Are you really sure you don't want to do it with an optional configmap?

Reply:
"This is fine" 😀